--- origsrc/gst-ffmpeg-0.10.13/configure.ac	2013-06-25 16:08:48.529868400 -0500
+++ src/gst-ffmpeg-0.10.13/configure.ac	2013-06-25 16:35:05.627073200 -0500
@@ -341,7 +341,7 @@ else
       embffmpeg_configure_args="$embffmpeg_configure_args --disable-mmx --disable-altivec"
       ;;
     mingw32*)
-      embffmpeg_configure_args="$embffmpeg_configure_args --enable-memalign-hack"
+      embffmpeg_configure_args="$embffmpeg_configure_args --enable-memalign-hack --enable-w32threads"
       WIN32_LIBS="-lws2_32"
       ;;
     *)
